src/validators.js

```diff
@@ -11,14 +11,20 @@ const validateEmail = (email) => {
   })
 }
 
-const formatDate = (date) => {
-  const splitDate = date.split('-')
-  const reshuffle = splitDate[2] +"-"+ splitDate[1] + "-"+ splitDate[0];
-  return reshuffle;
+
+const make_names_array = (obj, which_names) => {
+  var arrayObj = obj.reduce(function(acc, el){
```

Comment:
just a naming thing: `obj` doesn't sound like something you should be able to reduce on. Maybe call it something better?